client/src/modules/cohort/model/actionTypes.js

```diff
@@ -1,10 +1,25 @@
 export const CohortActionTypes = Object.freeze({
-  CREATE_COHORT_FAILURE: 'cohort/CREATE_COHORT_FAILURE',
-  CREATE_COHORT_REQUEST: 'cohort/CREATE_COHORT_REQUEST',
-  CREATE_COHORT_SUCCESS: 'cohort/CREATE_COHORT_SUCCESS',
+  ARCHIVE_FAILURE: 'cohort/ARCHIVE_FAILURE',
+  ARCHIVE_REQUEST: 'cohort/ARCHIVE_REQUEST',
+  ARCHIVE_SUCCESS: 'cohort/ARCHIVE_SUCCESS',
+  CREATE_FAILURE: 'cohort/CREATE_FAILURE',
+  CREATE_REQUEST: 'cohort/CREATE_REQUEST',
+  CREATE_SUCCESS: 'cohort/CREATE_SUCCESS',
+  DELETE_FAILURE: 'cohort/DELETE_FAILURE',
+  DELETE_REQUEST: 'cohort/DELETE_REQUEST',
+  DELETE_SUCCESS: 'cohort/DELETE_SUCCESS',
+  FETCH_ARCHIVED_META_DATA_FAILURE: 'cohort/FETCH_ARCHIVED_META_DATA_FAILURE',
+  FETCH_ARCHIVED_META_DATA_REQUEST: 'cohort/FETCH_ARCHIVED_META_DATA_REQUEST',
+  FETCH_ARCHIVED_META_DATA_SUCCESS: 'cohort/FETCH_ARCHIVED_META_DATA_SUCCESS',
+  FETCH_DATA_FAILURE: 'cohort/FETCH_DATA_FAILURE',
```

Comment:
consider renaming to cohort/FETCH_DETAILS  which is easy and clear to understand that in a first step you are fetching META-DATA of cohort, and then DETAILS of cohort. 
Fetching DATA for me is confusing because I don't know which kind of data you're fetching? Lists for cohorts?? or Extra DETAILS for cohort??